source/operations/network-encryption.rst

```diff
@@ -64,6 +64,12 @@ Enabling TLS
       If you have a custom Subject Alternative Name (SAN) certificate that is *not* also a wildcard cert, the TLS certificate SAN **must** apply to the hostname for its parent node.
       Without a wildcard, the SAN must match exactly to be able to connect to the tenant.
 
+   Certificate Management with cert-manager
+   ----------------------------------------
+
+   Rather than then MinIO Operator managing certificates, you can configure the deployment to use `cert-manager <https://cert-manager.io/>`__.
```

Comment:
```suggestion
   Rather than the MinIO Operator managing certificates, you can configure the deployment to use `cert-manager <https://cert-manager.io/>`__.
```